README.md

```diff
@@ -1,34 +1,67 @@
+
+The Amazon Chime SDK React Component Library supplies client-side state management and reusable UI components for common web interfaces used in audio and video conferencing applications, including: video tile grids, microphone activity indicators, and call controls. All components come with a simple, modern design, and can be used as-is or restyled with a custom theme. In addition to UI components, the library leverages Reacts' state management tools such as Providers and Hooks to connect to the Amazon Chime SDK and pass data to the UI layer, simplifying state synchronization so that developers can concentrate on building engaging experiences.
+
+The Amazon Chime React Component Library can be added to your web application codebase using npm.
+
+Documentation on these components and how to use them can be found on [Chime SDK React Component Library Documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/).
+
+### To get started, see the following resources:
+
+[Amazon Chime SDK](https://aws.amazon.com/chime/chime-sdk/)
+
+[Amazon SDK Pricing](https://aws.amazon.com/chime/pricing/#Chime_SDK_)
+
+[Supported Browsers](https://docs.aws.amazon.com/chime/latest/dg/meetings-sdk.html#mtg-browsers)
+
+[Amazon Chime API Reference](https://docs.aws.amazon.com/chime/latest/APIReference/Welcome.html)
+
+[Amazon Chime SDK for JavaScript Documentation](https://aws.github.io/amazon-chime-sdk-js/#amazon-chime-sdk-for-javascript)
 
-[Chime SDK UI Component Library Documentation](https://aws.github.io/amazon-chime-sdk-component-library-react/)
 [Migration to V2](https://aws.github.io/amazon-chime-sdk-component-library-react/?path=/docs/migration-to-v2--page)
 
-<a href="https://www.npmjs.com/package/amazon-chime-sdk-component-library-react"><img src="https://img.shields.io/npm/v/amazon-chime-sdk-component-library-react?style=flat-square"></a>
-<a href="https://github.com/aws/amazon-chime-sdk-component-library-react"><img src="https://github.com/aws/amazon-chime-sdk-component-library-react/workflows/CI%20Workflow/badge.svg"></a>
+## Examples
+
+[Chime SDK Meeting Demo](https://github.com/aws/amazon-chime-sdk-component-library-react/tree/master/demo/meeting)
+
+[Chat Demo](https://github.com/aws/amazon-chime-sdk-component-library-react/tree/master/demo/chat)
+
+## Installation and Development
 
-This contains reusable components written with React, TypeScript and styled components to be used to create UIs with Chime SDK JS.
+If you are adding this library to your existing application, add amazon-chime-sdk-component-library-react and the necessary peer dependencies to your project
+
+```
+npm install --save amazon-chime-sdk-component-library-react amazon-chime-sdk-js styled-components styled-system 
+```
 
-## To genereate dependencies
+If you have cloned this repo, then execute
```

Comment:
Would it help to separate this into two commands?

### Clone this repo
```
git clone https://github.com/aws/amazon-chime-sdk-component-library-react.git
```

### Generate dependencies
```
npm install
```

Reply:
i merged them into one